experimental/framework/benchmark/README.md

```diff
@@ -0,0 +1,30 @@
+# @fluid-experimental/benchmark
```

Comment:
Could this be in the @fluid-internal scope? Is it intended to be used outside the repo?

Reply:
I would like to use it outside of this repo. We have quite a lot of usage of this package in a private repo (including testing our app powered by experimental/tree), and I would like to just use the copy from fluid and delete our private one.

This does remind me though: there is another approach we could take here: This package could be published separately from fluid framework (ex: in a different repo), and then used by it. It may make sense to consider that option either now, and/or when/if we decide to make it non-experimental.

Reply:
> there is another approach we could take here: This package could be published separately from fluid framework (ex: in a different repo), and then used by it.

I like that option, because it makes "ownership" clearer and conceptually this package isn't tied to anything else in the repo. That said, it's fine to commit it here for now and we can discuss that if needed in the future during the experimental --> ready transition.